drafts/te-types-update/draft-ietf-teas-rfc8776-update.md

```diff
@@ -497,23 +425,23 @@ For the following URIs in the "IETF XML Registry" {{?RFC3688}}, IANA has updated
       XML: N/A, the requested URI is an XML namespace.
 ~~~~
 
-This document also adds updated YANG modules to the "YANG Module
+This document also requests IANA to add the following YANG modules to the "YANG Module
```

Comment:
to add or to update?

Reply:
Existing registrations are not updated, bit new ones are added even for bis documents. Please check https://www.iana.org/assignments/yang-parameters/yang-parameters.xhtml.